manifests/0000_26_cloud-controller-manager-operator_14_credentialsrequest-azure.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: openshift-azure-cloud-controller-manager
```

Comment:
Do we normally prefix these with openshift? Can we compare with the MAPI ones?

Reply:
Yes, all credential requests have `openshift` prefix